runtime/lib/backend/openqasm/braket_local_qubit.toml

```diff
@@ -1,97 +1,84 @@
-schema = 1
-
-[device]
-name = "braket.aws.qubit"
-
-[operators]
-# Observables supported by the device
-observables = [
-	"PauliX",
-	"PauliY",
-	"PauliZ",
-	"Hadamard",
-	"Hermitian",
-	"Tensor",
-]
+schema = 2
 
 # The union of all gate types listed in this section must match what
 # the device considers "supported" through PennyLane's device API.
-[[operators.gates]]
-native = [
-        "ISWAP",
-        "PSWAP",
-	"Hadamard",
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "S",
-        "T",
-        "CNOT",
-        "CZ",
-        "SWAP",
-        "PhaseShift",
-        "RX",
-        "RY",
-        "RZ",
-        "CSWAP",
-	"QubitUnitary",
-	"MS",
-	"CY",
-	"SX",
-	"AAMS",
-	"ECR",
-	"GPi",
-	"GPi2",
-	"CPhaseShift00",
-	"CPhaseShift01",
-	"CPhaseShift10",
-	"Identity",
-	"IsingXX",
-	"IsingXY",
-	"IsingYY",
-	"IsingZZ",
-	"Toffoli",
-	"ControlledPhaseShift",
-]
+[operators.gates.native]
+
+ISWAP = { }
+PSWAP = { }
+Hadamard = { }
+PauliX = { }
+PauliY = { }
+PauliZ = { }
+S = { }
+T = { }
+CNOT = { }
+CZ = { }
+SWAP = { }
+PhaseShift = { }
+RX = { }
+RY = { }
+RZ = { }
+CSWAP = { }
+QubitUnitary = { }
+MS = { }
+CY = { }
+SX = { }
+AAMS = { }
+ECR = { }
+GPi = { }
+GPi2 = { }
+CPhaseShift00 = { }
+CPhaseShift01 = { }
+CPhaseShift10 = { }
+Identity = { }
+IsingXX = { }
+IsingXY = { }
+IsingYY = { }
+IsingZZ = { }
+Toffoli = { }
+ControlledPhaseShift = { }
 
 # Operators that should be decomposed according to the algorithm used
 # by PennyLane's device API.
 # Optional, since gates not listed in this list will typically be decomposed by
 # default, but can be useful to express a deviation from this device's regular
 # strategy in PennyLane.
-decomp = []
+[operators.gates.decomp]
 
 # Gates which should be translated to QubitUnitary
-matrix = [
-]
+[operators.gates.matrix]
+
+[operators.observables]
+# Observables supported by the device
+PauliX = {}
+PauliY = {}
+PauliZ = {}
+Hadamard = {}
+Hermitian = {}
+# Tensor,
+Projector = {}
+Sprod = {}
+Hamiltonian = { condition = [ "analytic" ] }
+Sum = {}
+Prod = {}
 
 [measurement_processes]
-exactshots = [
-	"Expval",
-	"Var",
-	"Probs",
-	"State",
-]
-finiteshots = [
-	"Expval",
-	"Var",
-	"Probs",
-	"Sample",
-	"Counts",
-]
+
+Expval = {}
+Var = {}
+Probs = {}
+Sample = {}
```

Comment:
Note sampling also only works when a shot number is specified 😅 (needs updating in all TOMLs)

Reply:
Oh thanks. I mixed up Sample and State! State is analytic, as I see it. Fixing..

Reply:
True, state likely needs to be marked analytic only as well 👍